matdeeplearn/preprocessor/processor.py

```diff
@@ -267,7 +280,22 @@ def get_data_list(self, dict_structures, y):
         n_structures = len(dict_structures)
         data_list = [Data() for _ in range(n_structures)]
 
-        logging.info("Getting torch_geometric.data.Data() objects.")
+        logging.info(
+            "Getting torch_geometric.data.Data() objects and applying transforms."
+        )
+
+        # saving line graph attributes through transforms
+        transforms_list = []
+
+        if not self.otf:
+            for transform in self.transforms:
+                try:
+                    transforms_list.append(registry.get_transform_class(transform))
+                except KeyError:
+                    raise KeyError("No such transform found for '{transform}'")
```

Comment:
You actually don't need this code wrapped in a try/except clause because the registry code already handles those errors, so instead you can just do:
```suggestion
                transforms_list.append(registry.get_transform_class(transform))
```